chain_state.proto

```diff
@@ -1,4 +1,4 @@
-syntax = "proto3";
+syntax = "proto3"; # TODO delete this file?
```

Comment:
I was going to delete this file and all lines of code related to it, but then I figured I'd ask your approval first

Reply:
yes I think we're not using it anymore, it's safe to delete